packages/perseus/src/util/graphie.ts

```diff
@@ -1086,18 +1082,20 @@ export class Graphie {
     }
 
     svgPath = (points: (Coord | true)[], alreadyScaled?: boolean) => {
-        return $.map(points, (point, i) => {
-            if (point === true) {
-                return "z";
-            }
-            const scaled = alreadyScaled ? point : this.scalePoint(point);
-            return (
-                (i === 0 ? "M" : "L") +
-                KhanMath.bound(scaled[0]) +
-                " " +
-                KhanMath.bound(scaled[1])
-            );
-        }).join("");
+        return points
+            .map((point, i) => {
+                if (point === true) {
+                    return "z";
```

Comment:
**observation**: It seems like you'd only ever want this as the last point (or can SVG paths close the shape more than once?).  Anyways... the code isn't changing in this PR. :)